app/aggregates/deciding/decision.rb

```diff
@@ -128,11 +129,11 @@ def send_to_provider(user_id:, application_id:)
     end
 
     def update_from_maat(maat_attributes)
-      decision = Decisions::Draft.new(maat_attributes)
+      decision = Decisions::Draft.new(maat_attributes.except(:overall_result))
```

Comment:
Previously we cached the overall_result sent by MAAT for debugging purposes. We ignore this value from the event data.